Package.swift

```diff
@@ -77,10 +84,27 @@ let package = Package(
 
                 // opt-out of these backends (using metal)
                 "mlx/mlx/backend/no_metal",
-                "mlx/mlx/backend/accelerate",
+                "mlx/mlx/backend/no_cpu",
+
+                "mlx/mlx/backend/common/default_primitives.cpp",
+
+                // this uses neon code and will not build on x86 (e.g. via Release).
+                // see mlx-conditional/accelerate-softmax.cpp
+                "mlx/mlx/backend/accelerate/softmax.cpp",
```

Comment:
Special handling for building this on x86 (which we don't want, but Release builds will give us by default)